src/nearService.ts

```diff
@@ -0,0 +1,27 @@
+import { connect } from 'near-api-js';
+import { MergeKeyStore } from 'near-api-js/lib/key_stores';
+import { configService } from './config.service';
+
+const config = {
+  networkId: configService.getNetworkId(),
+  nodeUrl: configService.getNodeUrl(),
+  headers: {},
+  keyStore: new MergeKeyStore([]), // Keystore can be empty since we don't send any transactions
+};
+
+async function initializeNearService() {
```

Comment:
This function name is misleading, as it does not instantiate a NestJS injectable service, but an instance of a NEAR connection.